kubeflow/tf-job/prototypes/tf-job-operator.jsonnet

```diff
@@ -0,0 +1,26 @@
+// @apiVersion 0.1
```

Comment:
Do you think moving TFJob operator into the TFJob package makes sense? I'm uncertain what the proper organization of packages should be (see #106).

Its convenient for users not to install additional packages. Also, I think we'd like the all prototype to still install the TFJob operator. ksonnet doesn't handle dependencies between packages very well.

I'm not sure it makes sense to have a package just for TFJob. It might make sense to have a package for TensorFlow and put all the components for TensorFlow into that package.

What do you think?

Reply:
Ah, I see. Seems I was a little overzealous in breaking out the TFJob operator. ;-)
I do still think it's important to distinguish it from the "user" TFJob prototype.
So, I would prefer to name it one of:
* tf-job-operator
* tf-job-admin
* tf-job-service
* etc.

Reply:
tf-job-operator seems like a good name to me for the prototype.